subsys/mgmt/osdp/src/osdp_cp.c

```diff
@@ -627,6 +627,14 @@ static inline void cp_set_state(struct osdp_pd *pd, enum osdp_cp_state_e state)
 	CLEAR_FLAG(pd, PD_FLAG_AWAIT_RESP);
```

Comment:
In commit `osdp/mgmt: CP: Reset channel before sending a command ` you probably meant to say `mgmt/osdp: CP: Reset channel before sending a command` in commit subject.

Reply:
Yes. Fixed, thanks.